ios/mobileStackRuntime/Info.plist

```diff
@@ -33,7 +33,7 @@
 			<string></string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>celo</string>
+				<string>mobileStackRuntime</string>
```

Comment:
Do we want to use `mobileStackRuntime://...` for deep links? This feels too long.
Ideally this should be controlled from an env variable so it can be used as a single source of truth in the iOS and Android projects.

I think we missed when we put `DEEPLINK_PREFIX` in `src/config.ts`